html/partials/config_files.html.haml

```diff
@@ -8,50 +8,56 @@
         :title => "Config Files",
         :count => "#{count} #{Machinery.pluralize(count, "file")}"
       .row.scope_content.collapse.in
-        - config_files.files.each do |file|
-          - if file.diff
-            .diff{ style: "display: none", "data-config-file-diff" => file.diff.file }
-              %span.diff-summary
-                #{file.diff.additions} additions & #{file.diff.deletions} deletions
-              %table.diff
-                - file.diff.lines.each do |line|
-                  %tr{ class: line.type }
-                    %td.active.linenumber= line.original_line_number
-                    %td.linenumber= line.new_line_number
-                    %td
-                      = line.content
-        .col-md-1
-        .col-md-11
-          %table.table.table-striped.table-hover.table-condensed.files-table.filterable
-            %thead
-              %tr
-                %th Name
-                %th.medium Package Name
-                %th.medium Package Version
-                %th.medium Changes
-                %th.small Mode
-                %th.small User
-                %th.small Group
-            %tbody
-              - config_files.files.each do |file|
+        .col-xs-1
+        .col-xs-11
+        - if count == 0
```

Comment:
This needs to be indented I think:

![image](https://cloud.githubusercontent.com/assets/203813/11114127/9904ed0a-8922-11e5-8bd9-728966b4f784.png)